setup.cfg

```diff
@@ -64,6 +64,7 @@ xfail_strict = True
 filterwarnings =
     error:Sparse:FutureWarning
     error:The SparseArray:FutureWarning
+    always::ResourceWarning
```

Comment:
`default::ResourceWarning` might create less spam and probably catch the same mistakes. If I interpret `default` correctly, it will only create on warning instead of repeating it over and over (one test should fail, but not all test that use the failed function will fail). I might be wrong on that.

Reply:
can you leave this out?

Reply:
python doesn't show ResourceWarnings by default (irronically `-W default` is not the default). Instead of having` always::ResourceWarning` here (or `default:...`), it would also be possible to have `warnings.simplefilter("always", category=ResourceWarning)` within the context manager or in conftest.py.

Reply:
ok, but then how do we know this is working?

Reply:
I locally tested whether it errors on `open(...)` without a call to `close`. I will look into making a test that is expected to fail to  make sure that the line is not accidentally removed in the future.